### ISSUE TITLE
 Feedback rosetta BUG

### ISSUE THREAD

**Issue by tcqq** [ahmedaljazzar/LanguageSwitcher#22]
I  set the preferred language is (automatically detect the system language)，But if I changed the language of the phone system, and then open the program, the program language will not automatically switch

Code：     
        firstLaunchLocale = Locale.getDefault();
        supportedLocales = new HashSet<>();
        supportedLocales.add(Locale.CHINESE);
        supportedLocales.add(Locale.ENGLISH);
        supportedLocales.add(firstLaunchLocale);
        languageSwitcher = new LanguageSwitcher(this, firstLaunchLocale);
        languageSwitcher.setSupportedLocales(supportedLocales);

**Comment by iamjazzar** [comment]
Yes, I think this exists since it doesn't check the system language on every launch. Thanks for reporting this.
